app/src/main/java/com/monkeyteam/chimpagne/ui/MyEventsScreen.kt

```diff
@@ -89,13 +99,9 @@ fun MyEventsScreen(navObject: NavigationActions, myEventsViewModel: MyEventsView
                   }
                 } else {
                   items(uiState.createdEvents.values.toList()) { event ->
-                    ShortEventCard(
-                        title = event.title,
-                        date = timestampToStringWithDateAndTime(event.startsAtTimestamp),
-                        location = event.location.name,
-                        modifier = Modifier.testTag("a created event")) {
-                          navObject.navigateTo(Route.VIEW_DETAIL_EVENT_SCREEN + "/${event.id}")
-                        }
+                    EventCard(event = event, modifier = Modifier.testTag("a created event")) {
+                      navObject.navigateTo(Route.VIEW_DETAIL_EVENT_SCREEN + "/${event.id}")

```

Comment:
Choose between string concatenation styles and stick with it, at least on the same line